recipe/meta.yaml

```
@@ -18,7 +18,7 @@ source:
 build:
   number: {{ build_num }}
   # OOM in MSVC https://github.com/conda-forge/islpy-feedstock/pull/571
-  skip: true  # [win]
+  skip: true  # [win or py<=310]
```

This should probably be 
```suggestion
  skip: true  # [win or py<310]
```
?